$MirOS: ports/www/dillo/v2/patches/patch-dpi_https_c,v 1.3 2010/02/21 20:08:42 bsiegert Exp $
--- dpi/https.c.orig	Sun Jun  7 05:40:25 2009
+++ dpi/https.c	Sun Aug 23 17:05:23 2009
@@ -166,7 +166,7 @@ static void yes_ssl_support(void)
    }
 
    if (exit_error == 0){
-      snprintf(buf, 4095, "%s/.dillo/certs/", dGethomedir());
+      snprintf(buf, 4095, "%s/.etc/dillo/certs/", dGethomedir());
       if (SSL_CTX_load_verify_locations(ssl_context, NULL, buf )==0){
          MSG("Error opening user x509 certificate location\n");
          exit_error = 1;
@@ -646,13 +646,13 @@ static int save_certificate_home(X509 * 
    int retval = 1;
 
    /*Attempt to create .dillo/certs blindly - check later*/
-   snprintf(buf,4096,"%s/.dillo/", dGethomedir());
+   snprintf(buf,4096,"%s/.etc/dillo/", dGethomedir());
    mkdir(buf, 01777);
-   snprintf(buf,4096,"%s/.dillo/certs/", dGethomedir());
+   snprintf(buf,4096,"%s/.etc/dillo/certs/", dGethomedir());
    mkdir(buf, 01777);
 
    do {
-      snprintf(buf, 4096, "%s/.dillo/certs/%lx.%u",
+      snprintf(buf, 4096, "%s/.etc/dillo/certs/%lx.%u",
                dGethomedir(), X509_subject_name_hash(cert), i);
 
       fp=fopen(buf, "r");
